### ISSUE TITLE
Crash when I access to "Loaned" smart library 

### ISSUE THREAD

**Issue by GuillaumeHM** [mvz/alexandria-book-collection-manager#4]
Here is the trace I get :
```
$ alexandria 
/var/lib/gems/2.3.0/gems/alexandria-book-collection-manager-0.7.0/lib/alexandria/ui/ui_manager.rb:972:in `refresh_libraries': undefined method `name' for nil:NilClass
	 from /var/lib/gems/2.3.0/gems/alexandria-book-collection-manager-0.7.0/lib/alexandria/ui/sidepane.rb:134:in `block in setup_sidepane'
	 from /var/lib/gems/2.3.0/gems/gobject-introspection-3.0.9/lib/gobject-introspection/loader.rb:564:in `invoke'
	 from /var/lib/gems/2.3.0/gems/gobject-introspection-3.0.9/lib/gobject-introspection/loader.rb:564:in `block in define_method'
	 from /var/lib/gems/2.3.0/gems/alexandria-book-collection-manager-0.7.0/lib/alexandria/ui/ui_manager.rb:382:in `on_library_button_press_event'
	 from /var/lib/gems/2.3.0/gems/gobject-introspection-3.0.9/lib/gobject-introspection/loader.rb:114:in `invoke'
	 from /var/lib/gems/2.3.0/gems/gobject-introspection-3.0.9/lib/gobject-introspection/loader.rb:114:in `block in define_singleton_method'
	 from /var/lib/gems/2.3.0/gems/alexandria-book-collection-manager-0.7.0/lib/alexandria/ui.rb:47:in `start_gtk'
	 from /var/lib/gems/2.3.0/gems/alexandria-book-collection-manager-0.7.0/lib/alexandria/ui.rb:53:in `main'
	 from /var/lib/gems/2.3.0/gems/alexandria-book-collection-manager-0.7.0/lib/alexandria.rb:48:in `main'
	 from /var/lib/gems/2.3.0/gems/alexandria-book-collection-manager-0.7.0/bin/alexandria:64:in `<top (required)>'
	 from /usr/local/bin/alexandria:23:in `load'
	 from /usr/local/bin/alexandria:23:in `<main>'
/var/lib/gems/2.3.0/gems/glib2-3.0.9/lib/glib2.rb:41:in `exit': exit
	 from /var/lib/gems/2.3.0/gems/glib2-3.0.9/lib/glib2.rb:41:in `exit_application'
	 from /var/lib/gems/2.3.0/gems/gobject-introspection-3.0.9/lib/gobject-introspection/loader.rb:564:in `invoke'
	 from /var/lib/gems/2.3.0/gems/gobject-introspection-3.0.9/lib/gobject-introspection/loader.rb:564:in `block in define_method'
	 from /var/lib/gems/2.3.0/gems/alexandria-book-collection-manager-0.7.0/lib/alexandria/ui/ui_manager.rb:382:in `on_library_button_press_event'
	 from /var/lib/gems/2.3.0/gems/gobject-introspection-3.0.9/lib/gobject-introspection/loader.rb:114:in `invoke'
	 from /var/lib/gems/2.3.0/gems/gobject-introspection-3.0.9/lib/gobject-introspection/loader.rb:114:in `block in define_singleton_method'
	 from /var/lib/gems/2.3.0/gems/alexandria-book-collection-manager-0.7.0/lib/alexandria/ui.rb:47:in `start_gtk'
	 from /var/lib/gems/2.3.0/gems/alexandria-book-collection-manager-0.7.0/lib/alexandria/ui.rb:53:in `main'
	 from /var/lib/gems/2.3.0/gems/alexandria-book-collection-manager-0.7.0/lib/alexandria.rb:48:in `main'
	 from /var/lib/gems/2.3.0/gems/alexandria-book-collection-manager-0.7.0/bin/alexandria:64:in `<top (required)>'
	 from /usr/local/bin/alexandria:23:in `load'
	 from /usr/local/bin/alexandria:23:in `<main>'
```

I created a new smart library to do the same thing from scratch and it works fine. I can't delete the old one because I get the same crash when I right-click on it.
On the other smart libraries, the right-click menu shows a greyed button 'delete'.
Tell me if you need more informations. Thank you !
Guillaume

**Comment by GuillaumeHM** [comment]
$ alexandria 
/var/lib/gems/2.3.0/gems/alexandria-book-collection-manager-0.7.0/lib/alexandria/ui/ui_manager.rb:972:in `refresh_libraries': undefined method `name' for nil:NilClass
	 from /var/lib/gems/2.3.0/gems/alexandria-book-collection-manager-0.7.0/lib/alexandria/ui/sidepane.rb:134:in `block in setup_sidepane'
	 from /var/lib/gems/2.3.0/gems/gobject-introspection-3.0.9/lib/gobject-introspection/loader.rb:564:in `invoke'
	 from /var/lib/gems/2.3.0/gems/gobject-introspection-3.0.9/lib/gobject-introspection/loader.rb:564:in `block in define_method'
	 from /var/lib/gems/2.3.0/gems/alexandria-book-collection-manager-0.7.0/lib/alexandria/ui/ui_manager.rb:382:in `on_library_button_press_event'
	 from /var/lib/gems/2.3.0/gems/gobject-introspection-3.0.9/lib/gobject-introspection/loader.rb:114:in `invoke'
	 from /var/lib/gems/2.3.0/gems/gobject-introspection-3.0.9/lib/gobject-introspection/loader.rb:114:in `block in define_singleton_method'
	 from /var/lib/gems/2.3.0/gems/alexandria-book-collection-manager-0.7.0/lib/alexandria/ui.rb:47:in `start_gtk'
	 from /var/lib/gems/2.3.0/gems/alexandria-book-collection-manager-0.7.0/lib/alexandria/ui.rb:53:in `main'
	 from /var/lib/gems/2.3.0/gems/alexandria-book-collection-manager-0.7.0/lib/alexandria.rb:48:in `main'
	 from /var/lib/gems/2.3.0/gems/alexandria-book-collection-manager-0.7.0/bin/alexandria:64:in `<top (required)>'
	 from /usr/local/bin/alexandria:23:in `load'
	 from /usr/local/bin/alexandria:23:in `<main>'
/var/lib/gems/2.3.0/gems/glib2-3.0.9/lib/glib2.rb:41:in `exit': exit
	 from /var/lib/gems/2.3.0/gems/glib2-3.0.9/lib/glib2.rb:41:in `exit_application'
	 from /var/lib/gems/2.3.0/gems/gobject-introspection-3.0.9/lib/gobject-introspection/loader.rb:564:in `invoke'
	 from /var/lib/gems/2.3.0/gems/gobject-introspection-3.0.9/lib/gobject-introspection/loader.rb:564:in `block in define_method'
	 from /var/lib/gems/2.3.0/gems/alexandria-book-collection-manager-0.7.0/lib/alexandria/ui/ui_manager.rb:382:in `on_library_button_press_event'
	 from /var/lib/gems/2.3.0/gems/gobject-introspection-3.0.9/lib/gobject-introspection/loader.rb:114:in `invoke'
	 from /var/lib/gems/2.3.0/gems/gobject-introspection-3.0.9/lib/gobject-introspection/loader.rb:114:in `block in define_singleton_method'
	 from /var/lib/gems/2.3.0/gems/alexandria-book-collection-manager-0.7.0/lib/alexandria/ui.rb:47:in `start_gtk'
	 from /var/lib/gems/2.3.0/gems/alexandria-book-collection-manager-0.7.0/lib/alexandria/ui.rb:53:in `main'
	 from /var/lib/gems/2.3.0/gems/alexandria-book-collection-manager-0.7.0/lib/alexandria.rb:48:in `main'
	 from /var/lib/gems/2.3.0/gems/alexandria-book-collection-manager-0.7.0/bin/alexandria:64:in `<top (required)>'
	 from /usr/local/bin/alexandria:23:in `load'
	 from /usr/local/bin/alexandria:23:in `<main>'

**Comment by mvz** [comment]
Hi @GuillaumeHM thanks for your bug report! I'll look into it.

**Comment by mvz** [comment]
Hi @GuillaumeHM, can you run alexandria in debug mode like so:

```
alexandria -d
```

... and then make it crash again and post the output somewhere? That'll give me more information on what's leading up to the crash.

**Comment by GuillaumeHM** [comment]
Hi @mvz, I sent it to you by e-mail !

**Comment by mvz** [comment]
This has been fixed in version 0.7.1.
